devel/scons/Portfile

```diff
@@ -25,7 +25,8 @@ long_description    SCons is an Open Source software construction \
                     compiler caches such as ccache.
 homepage            http://www.scons.org/
 
-python.default_version 27
+python.versions     27 35 36 37 38
+python.default_version 37
```

Comment:
thanks for your contribution @kprussing !

It might make sense to switch to Python 3.7, but don't know whether Ryan's concerns in [Ticket 59303](https://trac.macports.org/ticket/59303) has been sufficiently investigated yet. 

Having said that, there is no point here in adding ```python.versions```: since the port is not in the "python" category or starts with "py-"; there will be no subports generated automatically. It's just a package that happens to use Python and indeed appears to be working with all Python versions you added here, but it will only use what is specified as ```python.default_version```. One possible way would perhaps be to add a "python2" variant, and use that for ports that would still need a PY2 version.

Reply:
It is not possible for a port to declare a dependency on a variant of another port (except through hacks like the active_variants portgroup) so that would not be ideal. But we are presuming that there are ports that use scons that require it to use python 2. I don't know if that is the case.